styles/app.scss

```diff
@@ -1,6 +1,72 @@
-// @import 'yoga';
-// @import 'normalize';
+@import 'normalize';
 
-// $fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
-// @import 'font-awesome';
+$fa-font-path: '//netdna.bootstrapcdn.com/font-awesome/4.3.0/fonts';
+@import 'font-awesome';
 @import 'box-sizing';
+@import 'container';
+
+
+$project-white: #fff;
+$projectgray: #808080;
+$projectlightgray: #d3d3d3;
+$projectgreen: #71C671;
+$projectblue: #009ACD;
+$projectfontsize: 2rem;
+
+
+// .container {
+//   height: 30rem;
+//   width: 20rem;
+//   margin: 2rem auto;
+//
+//
+//   border: 0.15rem solid $projectlightgray;
+//   border-radius: 0.5rem;
+//   padding: 2rem;
+// }
+
+.container i {
+  color: $projectlightgray;
+  font-size: 2rem;
+  display: inline-block;
+  position: relative;
+  top: -1rem;
```

Comment:
Instead of using `position: relative` here, try changing the margin of the parent container.